etc/test_delta_debug.sh

```diff
@@ -5,7 +5,8 @@
 # variable names such as $(false), unused variables, etc.
 set -ue -o pipefail
 
-cd "$(dirname $(readlink -f $0))/../flow"
+cd "$(dirname $(readlink -f $0))"
+cd ../../../flow
```

Comment:
Is this script exclusive to ORFS?

Reply:
it seems like it is mainly targeting ORFS

Reply:
To clarify: this integration test is testing the generic deltaDebug.py, which is independent of ORFS, using test-data from ORFS.

Reply:
Perhaps the integration test of deltaDebug.py could be moved to ORFS? Lots of OpenROAD integration testing happens in ORFS.